### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -18,40 +16,37 @@ ENV DEBIAN_FRONTEND=noninteractive \
...
-
-#Get urllib
-RUN wget -O - --no-check-certificate https://github.com/ValentinChCloud/urllib2_file/archive/master.tar.gz | tar -xz
-RUN mv urllib2_file-master urllib2_file; cd ./urllib2_file ; python setup.py test 
-RUN cd ./urllib2_file ; python setup.py build ; python setup.py install ;
-
-	
+# Get urllib
+# can you not simply pip install from github?  pip install https://github.com/user/repository/archive/branch.zip
+RUN wget -O - --no-check-certificate https://github.com/ValentinChCloud/urllib2_file/archive/master.tar.gz | tar -xz && \
```
Comment (bgruening) [inline Dockerfile]: can this be replaced by pip install from_github like in my comment?

**Comment by bgruening** [comment]
Those changes are untested, please have a look if they make sense....

**Comment by yvanlebras** [comment]
Thank you!!!!

**Comment by bgruening** [comment]
Can you let me know when this is finished?

**Comment by yvanlebras** [comment]
ok, in progress for now...

**Comment by yvanlebras** [comment]
ok, done! Build success!!!

**Pull request merged**